kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -346,7 +414,14 @@ private String sanitizeTableName(String tableName) {
    *         name if sanitization is enabled.
    */
   public String getTableFromTopic(String topic) {
-    return getBoolean(SANITIZE_TOPICS_CONFIG) ? sanitizeTableName(topic) : topic;
+    String tableName = null;
+    Map<String, String> map = getTopicsToTables();
```

Comment:
Re-evaluating the map every time is inefficient (albeit not a huge cost). getTableFromTopic is called by getRecordTable, which is called on every single record that KCBQ writes. I think the map should be resolved once, and re-used.